registry/storage/filewriter_test.go

```diff
@@ -165,8 +165,9 @@ func TestSimpleWrite(t *testing.T) {
 
 func TestBufferedFileWriter(t *testing.T) {
 	ctx := context.Background()
-	writer, err := newFileWriter(ctx, inmemory.New(), "/random")
 
+	writerp, err := newFileWriter(ctx, inmemory.New(), "/random")
+	writer := *writerp
```

Comment:
That's right, however, it this test case I do not pas a writer to any function. I only call functions on the writer which as far as I understand are all by reference.